x-pack/solutions/security/plugins/kubernetes_security/public/components/container_name_widget/styles.ts

```diff
@@ -44,7 +44,7 @@ export const useStyles = () => {
       borderRadius: euiTheme.border.radius.small,
       border: euiTheme.border.thin,
       bottom: '-25px',
-      boxShadow: `0 ${size.xs} ${size.xs} ${transparentize(euiTheme.colors.shadow, 0.04)}`,
+      boxShadow: `0 ${size.xs} ${size.xs} ${transparentize(euiTheme.colors.shadow, 0.04)}`, // TODO: replace transparentize with color token
```

Comment:
k8s_security should be decommissioned in 9.0 and serverless, therefore not touching, but leaving a comment in case the dashboard will be revived in the future